pandas/core/tools/numeric.py

```diff
@@ -130,6 +130,19 @@ def to_numeric(arg, errors="raise", downcast=None):
     1    2.1
     2    3.0
     dtype: Float32
+
+    Datetime dTypes is supported
+
+    >>> s = pd.to_numeric(datetime(2021, 8, 22), errors="coerce")
+    0 20210822
+    dtype: Int64
+    >>> s = pd.to_numeric(pd.NaT, errors="coerce")
+    nan
+    >>> s = pd.to_numeric(pd.Series(datetime(2021, 8, 22)), errors="coerce")
+    0 20210822
+    dtype: Int64
+    >>> s = pd.to_numeric(pd.Series(pd.NaT), errors="coerce")
+    NaN
```

Comment:
hmm. does the doctest pass on this? the output of an assignment should be None and nothing displayed?

Reply:
Yes , Nothing will display

Reply:
please let the question asker mark as resolved